tests/unit/dojson/test_dojson_utils.py

```diff
@@ -87,3 +88,89 @@ def test_remove_duplicates_from_list_of_dicts_preserving_order():
     result = utils.remove_duplicates_from_list_of_dicts(list_of_dicts_with_duplicates)
 
     assert expected == result
+
+
+def test_parse_conference_address():
+    addresses = [
+        'Waltham, Mass.',
+        'Dubna, USSR',
+        'Austin, Tex.',
+        'Amsterdam, Netherlands',
+        ''
+    ]
+
+    expected = [
+        {'city': None, 'country_code': 'US', 'latitude': None,
+         'longitude': None, 'original address': 'Waltham, Mass.',
+         'state': 'US-MA'},
+        {'city': None, 'country_code': 'SU', 'latitude': None,
+         'longitude': None, 'original address': 'Dubna, USSR',
+         'state': None},
+        {'city': None, 'country_code': 'US', 'latitude': None,
+         'longitude': None, 'original address': 'Austin, Tex.',
+         'state': 'US-TX'},
+        {'city': None, 'country_code': 'NL', 'latitude': None,
+         'longitude': None, 'original address': 'Amsterdam, Netherlands',
+         'state': None},
+        {'city': None, 'country_code': None, 'latitude': None,
+         'longitude': None, 'original address': '',
+         'state': None},
+    ]
+
+    results = [utils.parse_conference_address(address) for address in addresses]
+
+    assert expected == results
+
+
+def test_parse_institution_conference_address():
+    addresses = [
+        {'address': None, 'city': 'Beijing', 'state_province': None,
+         'country': 'China', 'postal_code': '123-CFG', 'country_code': None},
+        {'address': 'Tuscaloosa, AL 35487-0324', 'city': 'Tuscaloosa',
+         'state_province': 'AL', 'country': None,
+         'postal_code': 'PO Box 870324', 'country_code': None},
+    ]
+
+    expected = [
+        {'city': 'Beijing', 'country': 'China', 'country_code': 'CN',
+         'original_address': None, 'postal_code': '123-CFG', 'state': None},
+        {'city': 'Tuscaloosa', 'country': None, 'country_code': 'US',
+         'original_address': ('Tuscaloosa, AL 35487-0324',),
+         'postal_code': 'PO Box 870324', 'state': 'US-AL'}
+    ]
+
+    results = [
+        utils.parse_institution_address(**address) for address in addresses
+    ]
+
+    assert expected == results
+
+
+def test_classify_field():
+    """Test research field code mapping from arXiv to Inspire."""
+    raw_field_codes = ['HEP-TH', 'Gravitation and Cosmology', 'Super Cool Physics']
+    expected = ['Theory-HEP', 'Gravitation and Cosmology', 'Other']
+    result = [utils.classify_field(field) for field in raw_field_codes]
+
+    assert expected == result
+
+def test_classify_field_not_a_value():
+    """Test research field code mapping from arXiv to Inspire.
+
+    Case where input value is equivalent to None."""
+    raw_field_codes = [None, '']
+    expected = [None, None]
+    result = [utils.classify_field(field) for field in raw_field_codes]
+
+    assert expected == result
+
+def test_classify_field_not_a_string():
```

Comment:
Yes, PEP8 checking is disabled in tests: https://github.com/inspirehep/inspire-next/blob/f2bf1c45920b43f52a3002406ae3351b364f762f/pytest.ini#L27-L28